### PR TITLE
{tools}[gfbf/2023a] PyEXR v0.3.10

### PR DESCRIPTION
**Pull request by jfgrimm**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by migueldiascosta** [comment]
@boegelbot please test @ generoso

**Comment by migueldiascosta** [comment]
if this is required specifically for toolchain version 2023a, with OpenEXR 3.1.7, lgtm

it looks like the python bindings are included in OpenEXR itself from version 3.2.0, so an easyconfig for PyEXR using that toolchain version could probably be simplified (but I suppose that would also require adding e.g. `-DOPENEXR_BUILD_PYTHON=ON` to the existing OpenEXR-3.2.0-GCCcore-13.2.0.eb easyconfig)

**Comment by jfgrimm** [comment]
@migueldiascosta I actually tried doing exactly that (adding `-DOPENEXR_BUILD_PYTHON=ON` to the GCCcore/13.2.0 based ec) and... didn't end up with any python bindings in the `OpenEXR` install dir.

**Comment by migueldiascosta** [comment]
@jfgrimm looks like stills are still in flux (https://github.com/AcademySoftwareFoundation/openexr/pull/1629)

what about leaving a comment for whoever picks up on PyEXR later to look into using the python bindings from OpenEXR (3.2.3?) itself?

**Comment by verdurin** [comment]
@jfgrimm thoughts on Miguel's suggestion?

File: easybuild/easyconfigs/p/PyEXR/PyEXR-0.3.10-gfbf-2023a.eb [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+toolchain = {'name': 'gfbf', 'version': '2023a'}
+
+dependencies = [
+    ('Python', '3.11.3'),
+    ('SciPy-bundle', '2023.07'),
+    ('OpenEXR', '3.1.7'),
+]
+
+use_pip = True
+
```
Comment (jfgrimm) [inline easybuild/easyconfigs/p/PyEXR/PyEXR-0.3.10-gfbf-2023a.eb:20]:
```suggestion

# note: for future versions, look at building the python bindings in the `OpenEXR` module directly instead:
# see https://github.com/AcademySoftwareFoundation/openexr/pull/1629

**Comment by jfgrimm** [comment]
@migueldiascosta @verdurin good idea, done

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
easybuild-c7.novalocal - Linux CentOS Linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/verdurin/c227bc8d8c6b5f32d4febc2955cd6720 for a full test report.

**Comment by verdurin** [comment]
@boegelbot please test @ generoso

**Comment by verdurin** [comment]
@boegelbot please test @ jsc-zen3

**Review by verdurin — APPROVED** [review]
Looks fine.

**Pull request merged**